routes/index.js

```diff
@@ -255,6 +255,43 @@ module.exports = function () {
 
   // ===== /end docs
 
+  // Awesome
+  router.get(['/awesome', '/awesome/*'], (req, res) => {
+    const BASE = 'https://raw.githubusercontent.com/datahubio/awesome/master/';
+    let path = urllib.parse(req.path).path;
+    let fullpath;
+    if (path == '/awesome'){
+      fullpath = BASE + 'index.md'
+    } else {
+      // get the page name from the path and create a full adress
+      fullpath = BASE + path.split('/')[2] + '.md';
+    }
+    //request raw page from github
+    fetch(fullpath)
+        .then(function(res) {
```

Comment:
4 space indentation. In any case would read better as:

```
fetch(fullpath).then(...)
```

And even better would be to use async style:

```
const resp = await fetch(fullpath)
const text = await resp.text()
```